src/chain/chain_accumulator.rs

```diff
@@ -9,8 +9,66 @@
 use super::{AccumulatingEvent, NetworkEvent, Proof, ProofSet, SectionInfoSigPayload};
 use crate::id::PublicId;
 use log::LogLevel;
-use std::collections::{BTreeMap, BTreeSet};
-use std::mem;
+use std::collections::{BTreeMap, BTreeSet, VecDeque};
+use std::{mem, rc::Rc};
+
+/// The threshold (number of unvoted votes) a node to be considered as unresponsive.
+pub const UNRESPONSIVE_THRESHOLD: usize = 48;
+/// The period (X consensued observations) after which node be considered as unresponsive.
+pub const UNRESPONSIVE_WINDOW: usize = 64;
```

Comment:
It's still unclear to me from these comments what these constants mean. How about a short description of how the unresponsiveness detection mechanism works, and then how the constants fit in the picture?

Reply:
How about:
```
An unresponsive node is detected by conunting how many (defined by UNRESPONSIVE_THRESHOLD) missed votes among the certain number (defined by UNRESPONSIVE_WINDOW) of recent consensused observations.
```

Reply:
Sounds better :+1: